### DISCUSSION TITLE
1.25.2 + Wear OS 2.15.4

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#1157] [General]
- **General**: Reverted to old QR code scanning
- **General**: 3 new icons
- **General**: Translation updates

<hr /><em>This discussion was created from the release <a href='https://github.com/jamie-mh/AuthenticatorPro/releases/tag/1.25.2'>1.25.2 + Wear OS 2.15.4</a>.</em>

**Comment by Zinoujoker** [comment]
Good keep up 🆙 👍 😅
